controllers/storagecluster/cephcluster_test.go

```diff
@@ -338,7 +338,11 @@ func TestStorageClassDeviceSetCreation(t *testing.T) {
 			assert.Equal(t, deviceSet.DataPVCTemplate, scds.VolumeClaimTemplates[0])
 			assert.Equal(t, true, scds.Portable)
 			assert.Equal(t, c.sc.Spec.Encryption.Enable, scds.Encrypted)
-			assert.Equal(t, getPlacement(c.sc, "osd-tsc"), scds.Placement)
+			if scds.Portable && c.topologyKey == "rack" {
+				assert.Equal(t, scds.Placement.TopologySpreadConstraints[0].WhenUnsatisfiable, corev1.UnsatisfiableConstraintAction("DoNotSchedule"))
```

Comment:
While I know it'd be a hassle, you should check the full structure `scds.Placement.TopologySpreadConstraints`, like `len() == 2`, etc..